src/dag-node-utils.js

```diff
@@ -0,0 +1,210 @@
+'use strict'
```

Comment:
if this is going to be the utils file, just keep the name of the file that was before